osh/builtin_meta.py

```diff
@@ -328,6 +332,46 @@ def Run(self, cmd_val):
         return 0
 
 
+class Error(vm._Builtin):
+
+    def __init__(self, expr_ev):
+        # type: (expr_eval.OilEvaluator) -> None
+        self.expr_ev = expr_ev
+
+    def Run(self, cmd_val):
+        # type: (cmd_value.Argv) -> int
+        blame = loc.Word(cmd_val.arg_locs[0])
```

Comment:
I think you can just use arg_locs[0] directly right?  that is a CompoundWord, which is a `loc_t`